packages/editor/src/components/parts/common/customfield/StartCustomFieldPart.tsx

```diff
@@ -9,9 +9,9 @@ type StartCustomFieldPartProps = {
 
 const StartCustomFieldPart = ({ customFields, updateCustomFields }: StartCustomFieldPartProps) => {
   return (
-    <PathCollapsible label='Custom Fields' defaultOpen={customFields.length > 0} path='customFields'>
+    <PathContext path='customFields'>
```

Comment:
I think you can move this context inside of the customfieldtable? and use there a pathcollapsible?